pettingzoo/test/max_cycles_test.py

```diff
@@ -6,21 +6,24 @@ def max_cycles_test(mod):
     parallel_env = mod.parallel_env(max_cycles=max_cycles)
 
     observations = parallel_env.reset()
-    dones = {agent: False for agent in parallel_env.agents}
+    terminations = {agent: False for agent in parallel_env.agents}
+    truncations = {agent: False for agent in parallel_env.agents}
     test_cycles = (
         max_cycles + 10
     )  # allows environment to do more than max_cycles if it so wishes
     for step in range(test_cycles):
         actions = {
             agent: parallel_env.action_space(agent).sample()
             for agent in parallel_env.agents
-            if not dones[agent]
+            if not (terminations[agent] or truncations[agent])
         }
-        observations, rewards, dones, infos = parallel_env.step(actions)
-        if all(dones.values()):
+        observations, rewards, terminations, truncations, infos = parallel_env.step(
+            actions
+        )
+        if all(terminations.values() or truncations.values()):
```

Comment:
I don't think this works.

Reply:
Agreed. `all([x or y for x, y in zip(terminations.values(), truncations.values())])` should work.